applications/projects/Modeler/lib/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(SofaModeler)
 
-find_package(TinyXML REQUIRED)
+sofa_find_package(TinyXML REQUIRED)
```

Comment:
Note (FYI): this works but is not necessary because `SOFAMODELER_HAVE_TINYXML` is not needed in SofaModelerConfig.cmake.in (CMake level propagation) nor in config.h.in (C++ level propagation).

Reply:
Ha I see, I wasn't sure here since other packages that depends on tinyxml were doing it with `sofa_find_package` and it wasn't clear to me what this function was doing.  I'll put back the `find_package` since the modeler always have tinyxml